cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/service/S3LogUploaderServiceTest.java

```diff
@@ -51,4 +52,40 @@ public void test_that_getPartition_works() {
     String expected = "partitioned/year=2018/month=01/day=29/hour=12";
     assertEquals(expected, actual);
   }
+
+  @Test
+  public void test_that_getPartition_fails() {
+    String fileName = "dummy";
+    String actual = s3LogUploader.getPartition(fileName);
+    String expected = "un-partitioned";
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void test_ingest_log_works() {
+    String fileName = "localhost-audit.2018-01-29_12-58.log.gz";
+    s3LogUploader.ingestLog(fileName);
+    String fileName1 = "localhost-audit.2019-01-29_12-58.log.gz";
+    s3LogUploader.ingestLog(fileName);
+    s3LogUploader.ingestLog(fileName1);
+    //    try {
```

Comment:
Can we either uncomment or remove this test

Reply:
ok thanks , i will do that

Reply:
This test is still here, can you uncomment or remove this test?